### PR TITLE
optimize chart readme

### PR DESCRIPTION
**Pull request by huiwq1990**
**What this PR does / why we need it**:

1) As upgrade CustomResourceDefinition in #1296 ,  kubefed only support 1.16+
2) controllermanager docker image split into controller image and webhook image 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by makkes — APPROVED** [review]

**Comment by makkes** [comment]
/lgtm

Thanks a lot, @huiwq1990.

File: charts/kubefed/README.md [hunk 1]
```diff
@@ -98,14 +98,19 @@ chart and their default values.
...
+| controllermanager.webhook.repository           | Repo of the KubeFed image.                                                                                                                                                         | quay.io/kubernetes-multicluster |
+| controllermanager.webhook.image                | Name of the KubeFed image.                                                                                                                                                         | kubefed                         |
+| controllermanager.webhook.tag                  | Tag of the KubeFed image.                                                                                                                                                          | canary                          |
+| controllermanager.controller.replicaCount      | Number of replicas for KubeFed controller manager.                                                                                                                                 | 1                          |
```
Comment (irfanurrehman) [inline charts/kubefed/README.md:112]: This seems like a repeat of line 107?

File: charts/kubefed/README.md [hunk 2]
```diff
@@ -98,14 +98,19 @@ chart and their default values.
...
+| controllermanager.controller.imagePullPolicy   | Image pull policy.                                                                                                                                                                 | IfNotPresent                          |
```
Comment (irfanurrehman) [inline charts/kubefed/README.md:113]: Did you mean `controllermanager.webhook.imagePullPolicy`?

**Review by irfanurrehman — CHANGES_REQUESTED** [review]
Thanks @huiwq1990 
I might be missing something, but it appears like there are duplicates. Please check and let me know.

**Comment by huiwq1990** [comment]
@irfanurrehman I have recheck my pr, and fix them.

**Comment by huiwq1990** [comment]
yes, my mistake.

**Comment by makkes** [comment]
We should put all options in (alphabetic?) order but that can be done in a subsequent PR.

/lgtm

File: charts/kubefed/README.md (hunk 1 above)
Comment (huiwq1990) [reply charts/kubefed/README.md]: yes, i have fix them.

**Comment by huiwq1990** [comment]
I prefer organize them by kubefed’s module(webhook,controller).  User know how to config them at first glance.

**Comment by irfanurrehman** [comment]
Thanks @huiwq1990 
/approve

**Review by irfanurrehman — APPROVED** [review]

**Pull request merged**